### PR TITLE
support for Ubuntu 20.04

### PR DESCRIPTION
**Pull request by lschierer**
#### Pull Request (PR) description
When I added a Ubuntu 20.04 node to my puppet master, it could not find the package name for the DNS plugin.  This fixed it.

### CODE REVIEW

**Comment by kenyon** [comment]
We would also need to update metadata.json to declare support for Ubuntu 20.04.

**Comment by lschierer** [comment]
I have updated that file now as well, I missed it before.

File: metadata.json
Comment (kenyon) [inline metadata.json:40]:
```suggestion
        "18.04",
```

**Review by kenyon — CHANGES_REQUESTED** [review]

File: metadata.json
Comment (lschierer) [reply metadata.json]: done

**Review by kenyon — COMMENTED** [review]
Thanks. Looks like the remaining work to get the tests to pass is in #235 and possibly other PRs.

**Comment by lschierer** [comment]
can we remove the needs work label, or is there more I should do?

**Comment by lschierer** [comment]
it looks like a maintainer has to approve the workflow now to see if the latest merge from upstream resolves any of the test failures.

**Comment by lschierer** [comment]
it appears I am now passing all checks, so the tests-fail tag added by https://github.com/voxpupuli/puppet-letsencrypt/pull/244#issuecomment-858544263 should be removed.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
@kenyon I think your concerns have been addressed, but you are listed as "requested changes" now. Any objections to merging this?

Also, we should squash this. We can do that while merging, but IMHO it's better for a cleaner git log.

**Comment by lschierer** [comment]
now squashed per request.

**Comment by ekohl** [comment]
Looks like something went wrong there because I see 18 commits. Can you use `git rebase`?

**Comment by lschierer** [comment]
I cannot figure out enough git rebase to clean this up. I'm going to close this and recreate it.

**Pull request closed without merging**
